spec/requests/admin/comments_spec.rb

```diff
@@ -0,0 +1,194 @@
+require 'rails_helper'
+
+RSpec.describe 'Comment moderation', type: :request do
+  before :each do
+    @admin = create :discussion_admin
```

Comment:
It doesn't look like `@admin` needs to be an instance variable. May be the case with some of these other `@instance_variables` defined in the `before` block.